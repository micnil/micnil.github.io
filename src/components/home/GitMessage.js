import React from 'react';
import styled from 'styled-components';
import CodeBlock from '../CodeBlock';
import Email from '../Email';

const Yellow = styled.span`
  color: rgb(255, 204, 0);
`;

const Grey = styled.span`
  color: rgb(174, 174, 170);
`;

const GitMessage = () => (
<CodeBlock>
  <Yellow>></Yellow><span>{` git log -1
`}</span><Grey>commit daff16378c079fb52c1bd27dc6f630988b0d222a</Grey><span>{`
Author: Michael Nilsson <`}</span><Yellow><Email /></Yellow><span>{`>
Date:   Sun Mar 3 19:20:58 2019 +0200

    [#2157] `}</span><Yellow>Add introduction to Michael Nilsson</Yellow><span>{` ðŸš€

    Explain that Michael is a Swedish software developer, that enjoys
    solving interesting problems both professionally and as a hobby.
    Promote his interests in new technology, and that he is driven by
    building innovative solutions. Mention that he has a strong 
    background in `}</span><Yellow>web technologies</Yellow><span>{`, `}</span><Yellow>.NET</Yellow><span>{` and `}</span><Yellow>computer graphics</Yellow><span>{`. 

    Also point out that he has a `}</span><Yellow>Master of Science in Media Technology</Yellow><span>{`.`}</span>
</CodeBlock>
);

export default GitMessage;