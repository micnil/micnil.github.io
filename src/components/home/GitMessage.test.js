import React from 'react';
import renderer from 'react-test-renderer';
import GitMessage from './GitMessage';

describe('GitMessage', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<GitMessage />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
