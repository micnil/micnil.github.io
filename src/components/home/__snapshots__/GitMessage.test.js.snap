// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitMessage renders correctly 1`] = `
<pre
  className="sc-bdVaJa cEOuNe"
>
  <code
    className="sc-bwzfXH gdMrTi"
  >
    <div>
      <span
        className="sc-htpNat iFhYwv"
        color="#ff5f56"
      />
      <span
        className="sc-htpNat kmdSYw"
        color="#ffbd2e"
      />
      <span
        className="sc-htpNat hMNoyt"
        color="#27c93f"
      />
    </div>
    <span
      className="sc-bxivhb lgObjd"
    >
      &gt;
    </span>
    <span>
       git log -1

    </span>
    <span
      className="sc-ifAKCX fizYCp"
    >
      commit daff16378c079fb52c1bd27dc6f630988b0d222a
    </span>
    <span>
      
Author: Michael Nilsson &lt;
    </span>
    <span
      className="sc-bxivhb lgObjd"
    >
      <span>
        michael
        <span
          style={
            Object {
              "display": "none",
            }
          }
        />
        @
        nilsson
        <span />
        .dev
      </span>
    </span>
    <span>
      &gt;
Date:   Sun Mar 3 19:20:58 2019 +0200

    [#2157] 
    </span>
    <span
      className="sc-bxivhb lgObjd"
    >
      Add introduction to Michael Nilsson
    </span>
    <span>
       ðŸš€

    Explain that Michael is a Swedish software developer, that enjoys
    solving interesting problems both professionally and as a hobby.
    Promote his interests in new technology, and that he is driven by
    building innovative solutions. Mention that he has a strong 
    background in 
    </span>
    <span
      className="sc-bxivhb lgObjd"
    >
      web technologies
    </span>
    <span>
      , 
    </span>
    <span
      className="sc-bxivhb lgObjd"
    >
      .NET
    </span>
    <span>
       and 
    </span>
    <span
      className="sc-bxivhb lgObjd"
    >
      computer graphics
    </span>
    <span>
      . 

    Also point out that he has a 
    </span>
    <span
      className="sc-bxivhb lgObjd"
    >
      Master of Science in Media Technology
    </span>
    <span>
      .
    </span>
  </code>
</pre>
`;
